features/gezag-minderjarige/1.1-ingezetene-BRP.feature

```diff
@@ -0,0 +1,122 @@
+#language: nl
+
+Functionaliteit: 1.1 - Staat persoon (minderjarige) als ingezetene in de BRP?
+
+
+    Achtergrond:
+      Gegeven de persoon 'P1' met burgerservicenummer '000000012'
+      * is meerderjarig
+      * heeft geslachtsnaam gevuld
+      En de persoon 'P2' met burgerservicenummer '000000024'
+      * is meerderjarig
+      * heeft geslachtsnaam gevuld
+      En persoon 'P1' heeft geregistreerd partnerschap met persoon 'P2'
+      | datum huwelijkssluiting/aangaan geregistreerd partnerschap (06.10) |
+      | gisteren - 20 jaar                                                 |
+
+
+  Regel: Als de gemeente van inschrijving is RNI kan het gezag van de persoon niet worden bepaald
+    # een persoon staat in de RNI ingeschreven wanneer gemeente van inschrijving (09.10) gelijk is aan 1999.
+    # een persoon staat in de BRP ingeschreven wanneer gemeente van inschrijving (09.10) ongelijk is aan 1999.
+
+    Scenario: minderjarige staat ingeschreven in RNI (route 1)
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is minderjarig
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
```

Comment:
ik zou ervoor kiezen om `vanaf de geboorteaangifte` in de stap beschrijving weglaten en dit standaard in de automation te laten invullen. Ook hier is het voor mij interessant om te weten of het uitmaakt wanneer een persoon als ouder wordt opgenomen in de BRP. Wordt de stap beschrijving dan

`heeft persoon 'P1' als ouder1 ná de geboorteaangifte`

of

`heeft persoon 'P1' als ouder1 16 jaar ná de geboorteaangifte`

of willen we

```
heeft persoon 'P1' als ouder1
| datum ingang familierechtelijke betrekking (62.10) |
| 16 jaar ná geboorteaangifte |

Reply:
ik vind het prima om de toevoeging " vanaf de geboorteaangifte" te verwijderen. Betekent wel dat de stap "heeft persoon 'P1' als ouder1' de datum ingang familierechtelijke betrekking standaard moet vullen met de geboortedatum van het kind.

Kunnen we hetzelfde doen met de stap "persoon 'P1' heeft geregistreerd partnerschap met persoon 'P2'"? Daar moet "datum huwelijkssluiting/aangaan geregistreerd partnerschap" standaard liggen vóór de geboortedatum van het kind/de kinderen.

Reply:
eerste paragraaf is ondertussen zo.

@fsamwel is  tweede paragraaf nog nodig? in verschillende features is het nu opgelost door e.g. is gehuwd `met de volgende gegevens` te gebruiken.